Jenkinsfile

```diff
@@ -74,6 +74,13 @@ pipeline {
                     sh '.ci/scripts/distribution/test-go.sh'
                 }
             }
+
+            post {
```

Comment:
What you mean by properly? This makes so the results of the go tests are posted as soon as they end instead of when all tests end. This makes it easier and faster to catch problems in the integration tests. Since before we weren't always building the zbctl binary, it happened frequently that it was outdated and some minor thing broke the integration tests. Now, not only we catch them more easily locally (although it's slower to run the IT tests) but if something fails on the CI we see it quicker as well. However, there is still [a bug](https://github.com/jstemmer/go-junit-report/issues/95) in the output that makes the '+' and '-' signs disappear from diffs, if that's what you meant